API.md

```diff
@@ -0,0 +1,325 @@
+# Secure Messaging API
+
+
+* `GET /messages`
+
+&mdash; When a list of messages is requested this returns a generic representation of the list of messages available.
+
+&mdash; Results can be filtered further by using query parameters:
+
+| `**Variable**` | `**Type**` | `**Example Value**` |
+| :---: | :---: | :---: |
+| cc (collection_case) | `string` | ACollectionCase |
+| ru_id | `string` | (not implemented)
+| survey | `string` | BRES |
+| label | `string` | INBOX/DRAFT/SENT |
+| ce (collection_exercise) | `string`| ACollectionExercise |
+| desc (descending) | `boolean` | True/False
```

Comment:
Default value for desc is 'True'. Also maybe be more descriptive as to what this toggle does? True = Most recent messages first.